File: eval/fixtures/legacy-directory-backed-spring-commerce/src/main/java/com/acme/legacy/orders/OrderFulfillmentService.java
package com.acme.legacy.orders;

public class OrderFulfillmentService {
  private final OrderFulfillmentRepository orderFulfillmentRepository = new OrderFulfillmentRepository();

  public OrderRepriceResult repriceOrder(String orderId) {
    return orderFulfillmentRepository.repriceFulfillmentBacklog(orderId);
  }
}

File: eval/fixtures/legacy-directory-backed-aspnet-commerce/src/Legacy/Orders/OrderFulfillmentService.cs
namespace Acme.Legacy.Orders;

public class OrderFulfillmentService
{
  private readonly OrderFulfillmentRepository _orderFulfillmentRepository = new OrderFulfillmentRepository();

  public OrderRepriceResult RepriceOrder(string orderId)
  {
    return _orderFulfillmentRepository.RepriceFulfillmentBacklog(orderId);
  }
}
